### PR TITLE
Add weekly update 2016-01-29

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-01-29.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The Node.js project will be releasing new versions across all of its active release lines early next week (possibly sooner, pending full impact assessment) to incorporate upstream patches from OpenSSL and some additional low-severity fixes relating to HTTP handling. Please read on for full details.
+
+See https://nodejs.org/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016/ for more information.
+
+### OpenSSL Impact Assessment
+
+OpenSSL versions 1.0.1r and 1.0.21 have been released, the announcement can be found here: https://mta.openssl.org/pipermail/openssl-announce/2016-January/000061.html
+
+Our team has made an assessment of the impact of the disclosed defects and concluded that there is no urgency in releasing patched versions of Node.js in response to this release. Therefore, we will be proceeding as planned and attempt to release new versions of each of our active release lines on or after
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md:21]: `on or after`? Shouldn't that have been `on or before`?

File: locale/en/blog/weekly-updates/weekly-update.2016-01-29.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - **[@bnb](https://github.com/bnb)**: [nomination post](http://bnb.im/blog/Individual-Membership-on-the-Board-of-Directors-for-Node-js.html), [Q/A thread](https://github.com/nodejs/membership/issues/26)
+  - **[@watson](https://github.com/watson)**: [nomination post](https://medium.com/@wa7son/why-i-m-running-for-the-node-js-foundation-board-of-directors-253bc2e3a834), [Q/A thread](https://github.com/nodejs/membership/issues/27)
+  - **[@feross](https://github.com/feross)**: [nomination post](http://feross.org/node-board/), [Q/A thread](https://github.com/nodejs/membership/issues/28)
+  - **[@pgte](https://github.com/pgte)**: [nomination post](https://gist.github.com/pgte/cfbf468202b35be78c66), [Q/A thread](https://github.com/nodejs/membership/issues/29)
+  - **[@ecowden](https://github.com/ecowden)**: [nomination post](https://medium.com/@evan.cowden/the-world-s-worst-resume-e0adf234baa0), [Q/A thread](https://github.com/nodejs/membership/issues/30)
+
+A ballot will be distributed to individual members on January 20th, with the election completed by January 30th.
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md]: It has already been distributed. So, can this be changed to "A ballot was distributed to individual members on January 20th, with the election completed by January 30th."

**Comment by JungMinu** [comment]
@thefourtheye Thanks, revised :)

File: locale/en/blog/weekly-updates/weekly-update.2016-01-29.md (hunk 1 above)
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md:21]: Nope, should be `on or after`. See https://github.com/nodejs/nodejs.org/blob/b3469dc2c4f1a45a5ac9ac09c9b7f02c5e42951e/locale/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016.md#update-29-jan-2016-openssl-impact-assessment

File: locale/en/blog/weekly-updates/weekly-update.2016-01-29.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The Node.js project will be releasing new versions across all of its active release lines early next week (possibly sooner, pending full impact assessment) to incorporate upstream patches from OpenSSL and some additional low-severity fixes relating to HTTP handling. Please read on for full details.
+
+See https://nodejs.org/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016/ for more information.
+
+### OpenSSL Impact Assessment
+
+OpenSSL versions 1.0.1r and 1.0.21 have been released, the announcement can be found here: https://mta.openssl.org/pipermail/openssl-announce/2016-January/000061.html
+
+Our team has made an assessment of the impact of the disclosed defects and concluded that there is no urgency in releasing patched versions of Node.js in response to this release. Therefore, we will be proceeding as planned and attempt to release new versions of each of our active release lines on or before
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md]: I think this was correct originally. Should be `on or after`. See https://github.com/nodejs/nodejs.org/blob/b3469dc2c4f1a45a5ac9ac09c9b7f02c5e42951e/locale/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016.md#update-29-jan-2016-openssl-impact-assessment

**Comment by JungMinu** [comment]
@Trott Thanks, updated :)

File: locale/en/blog/weekly-updates/weekly-update.2016-01-29.md (hunk 1 above)
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md:21]: But two paragraphs above, I saw "possibly sooner, pending full impact assessment". That is why I suggested "before".

Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-01-29.md:21]: Ah, I see! Now that the full impact assessment is done and a decision has been made, `on or after` seems to be the way it's going to happen.

**Pull request merged**